docs-sphinx/uproot3-to-4.rst

```diff
@@ -4,3 +4,79 @@ Uproot 3 → 4 cheat-sheet
 I'm collecting ideas for this cheat-sheet on the `Uproot 4 wiki <https://github.com/scikit-hep/uproot4/wiki>`__.
 
 If you think of something that would be good for me to write here, dump it there (doesn't need to be polished).
+
+Custom interpretations
+----------------------
+
+Whenever the automatic serialisation of uproot fails for whatever reason, a
+custom interpretation comes to the rescue. The concept remained the same but a
+few details have been changed. Below are two examples, the first showing how to
+pass a custom interpretation in ``uproot3`` and the second the same approach in
+``uproot4``.
+
+uproot3
+~~~~~~~
+
+.. code-block:: python
+
+    import uproot  # version 3
+    from skhep_testdata import data_path
+
+    f = uproot.open(data_path("uproot-issue124.root"))
+
+    tree = f["KM3NET_EVENT"]
+
+    snapshot_hits = tree["snapshotHits"].array(
+        uproot.asjagged(
+            uproot.astable(
+                uproot.asdtype(
+                    [
+                        ("dom_id", ">i4"),
+                        ("channel_id", "u1"),
+                        ("time", "<u4"),
+                        ("tot", "u1"),
+                    ]
+                )
+            ),
+            skipbytes=10,
+        )
+    )
+
+This will return an ``awkward0.JaggedArray``:
+
+.. code-block:: python
+    >>> snapshot_hits
```

Comment:
I'm pretty sure you need a gap between `.. code-block:: python` and the code. reStructuredText is finicky.

Reply:
Absolutely right, I always forget about that...